daml-lf/spec/daml-lf-1.rst

```diff
@@ -2672,6 +2672,12 @@ as described by the ledger model::
        ⇓ᵤ
      Ok (cid, tr) ‖ (st₁, keys₁)
 
+     cid ∉ dom(st₀)
```

Comment:
Do we need the `0` indices here and below?

Reply:
Removed!